paper/paper.md

```diff
@@ -0,0 +1,233 @@
+which simply requires the user to activate more threads in their Julia REPL. This is of
+particular importance since the transition analysis is sometimes performed on 10,000+
+timeseries [@ben-yami_uncertainties_2023], which can last hours with a serial code that is
+not optimized.
+
+## Reproducibility
+
+Some steps of a transition analysis involve random number generators, which need 
+to be handled with care in parallelized codes. This is done in TransitionsInTimeseries.jl, 
+which offers the possibility of seeding a random number generator by using the 
+keywordargument `rng`, as done in the example shown above. Furthermore, 
+TransitionsInTimeseries.jl follows the guidelines of semantic versionning which, 
+along with Julia's integrated package manager, ensures that the same code is used for both 
+results generation and peer reviewing.
+
+## Reliability
+
+In high-impact context mentioned above, errors are to be avoided with particular care.
+TransitionsInTimeseries.jl is therefore tested via continuous integration on a large test
+suite, thus providing a reliable research framework. Furthermore, a centralised code base
+implies that any new user is a new test, thus increasing the reliability of the code over
+time. Finally, the robustness of the results with respect to a parameter, e.g. the width
+of the sliding window, can be easily studied thanks to the simple syntax, thus contributing
+to the reliability of the results.
+
+## Versatility
+
+TransitionsInTimeseries.jl detaches the degrees of freedom that are available to the user
+from the analysis pipeline. Methods for detection and prediction can thus be applied equally
+well, since both typically rely on the computation of statistical measures over windows of
+the timeseries. This is of great importance, since a new prediction technique needs to first
+be tested on hindcasting tasks, which requires a reliable detection and timing
+of previous transitions. To illustrate this, a detection task can be performed by merely
+modifying the indicators, change metrics and window type of the code shown above:
+
+```julia
+# Here the data should not be detrended
+time, data = load_data()
+
+indicators = (nothing, nothing)
+change_metrics = (difference_of_mean(), difference_of_max())
+config = SlidingWindowConfig(indicators, change_metrics;
+    width_cha = 50, whichtime = midpoint)
+```
+
+This example showcases that steps of the analysis pipeline can be skipped alltogether, as
+done here for the computation of indicators. This is not needed here, since the change
+metric of the original time series is already sufficient to detect a transition by comparing
+the difference in mean and maximum values between the two halves of the sliding window.
+Furthermore, different ways of testing for significance can be interchangeably used,
+therefore expanding the versatility of the software beyond the surrogate analysis.
+
+## Extensibility
```

Comment:
This section misses that it is easy to add completely new ways of estimating changes, as in making new `IndicatorChangesConfig`.

Reply:
I am not sure to have fully understood what you mean but I added what I got :)